### ISSUE TITLE
"Folder as Workspace" loses color scheme on N++ window re-focus

### ISSUE THREAD

**Issue by terencemcdonnell** [notepad-plus-plus/notepad-plus-plus#10091]
### Description of the Issue

I have seen this bug appear and disappear multiple times over different releases.

Currently using 
### Steps to Reproduce the Issue

Seems to happen when N++ is put into background, and certain other apps get focus, then those apps are closed and N++ regains focus.  The WebEx video teleconferencing client is a consistent application that triggers this behavior.

I noticed that for a while, this issue would occur when using "Find in files" when closing the search dialog.

The "Folder as a workspace" pane loses it's background color, and turns grey, while the foreground of the pane (the folders, treeview, etc.) retain coloring.

If the folder treeview is expanded by clicking the plus icon, a portion of the pane will regain its coloring.

Refreshing the window does nothing.  Repainting the window (by losing/gaining focus) does nothing.

Minimizing and restoring the app resolves the issue, so I have a macro that does this, but it is a pain to always be fixing this during normal workflow.

I have tried this with/without plugins installed (used the N++ command line option to start the app with no plugins, and manually disabled the plugins by renaming the plugin folder) but this did not make a difference.

Am using "Vim Dark" color scheme.

Any assistance appreciated.

Love the product - thanks - Terence.

![n++_Issue-2021-06-30](https://user-images.githubusercontent.com/22206296/124148020-3de28080-da5d-11eb-9b02-2794688f31d2.jpg)
### Debug Information
Notepad++ v8.1   (64-bit)
Build time : Jun 17 2021 - 01:56:07
Path : C:\Users\E022019\Documents\Apps\NPP\notepad++.exe
Command Line : 
Admin mode : OFF
Local Conf mode : ON
Cloud Config : OFF
OS Name : Windows 10 Enterprise (64-bit) 
OS Version : 1809
OS Build : 17763.1971
Current ANSI codepage : 1252
Plugins : BigFiles.dll ComparePlugin.dll DSpellCheck.dll MarkdownViewerPlusPlus.dll mimeTools.dll NppConverter.dll NppExec.dll NppExport.dll NppFavorites.dll NPPJSONViewer.dll NppMarkdownPanel.dll NppTextFX.dll NppXmlTreeviewPlugin.dll PoorMansTSqlFormatterNppPlugin.dll PythonScript.dll _CustomizeToolbar.dll

**Comment by TontonJambon** [comment]
I have encountered the same issue when opening TightVNC. The issue was there on before I upgraded too and does not seem to be related at all with the dark mode (which I love by the way !). Expanding file tree or scrolling fix the color issue, as with a redraw issue.

![image](https://user-images.githubusercontent.com/6491892/126660416-1d4e8179-8a04-44e7-bbff-f63f611b9139.png)

![image](https://user-images.githubusercontent.com/6491892/126660543-39330664-9cd1-4176-ab5f-1f6fa36d74b9.png)

Here are my debug informations: 
Notepad++ v8.1.1   (32-bit)
Build time : Jul  1 2021 - 14:04:28
Path : C:\Program Files (x86)\Notepad++\notepad++.exe
Command Line : 
Admin mode : OFF
Local Conf mode : OFF
Cloud Config : OFF
OS Name : Windows 10 Enterprise (64-bit) 
OS Version : 2009
OS Build : 19042.1110
Current ANSI codepage : 1252
Plugins : mimeTools.dll NppConverter.dll NppExport.dll 

Love the product, thank you for developing it.
